README.md

```diff
@@ -37,7 +37,7 @@ And dozens upon dozens of smaller features.
 
 This is the current state of building the app. We're actively working to make this process much more streamlined!
 
-*You'll need Docker installed to run the app locally.*
+_You'll need Docker installed to run the app locally._
```

Comment:
I guess we can do this in a separate PR?

```suggestion
*You'll need Docker installed to run the app locally.*
```

Reply:
Dumb format on save, I think its fine.